client/components/data/media-list-data/index.jsx

```diff
@@ -26,6 +26,7 @@ module.exports = React.createClass( {
 
 	propTypes: {
 		siteId: React.PropTypes.number.isRequired,
+		source: React.PropTypes.string,
```

Comment:
The source parameter doesn't seem to be used in this component, neither it has a default value. Is something that you plan to use in upcoming PRs?